### PR TITLE
fix: using new pubsub-topic flag and updating version to v0.22.0-rc.0

### PR DESCRIPTION
**Pull request by gabrielmer**
- [x] Updating version to `v0.22.0-rc.0` in order to use TWN
- [x] Changing deprecated `--topic` flag for `--pubsub-topic`

### CODE REVIEW

**Review by jm-clius — APPROVED** [review]
LGTM! Thanks.

File: docker-compose.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ x-pg-exporter-env: &pg_exp_env
 # Services definitions
 services:
   nwaku:
-    image: ${NWAKU_IMAGE:-wakuorg/nwaku:v0.21.3}
+    image: ${NWAKU_IMAGE:-wakuorg/nwaku:v0.22.0-rc.0}
```
Comment (SionoiS) [inline docker-compose.yml:34]: @vpavlin said `NWAKU_IMAGE=docker.io/wakuorg/nwaku:v0.22.0-rc.0` does it matter?

```suggestion
    image: ${NWAKU_IMAGE=docker.io/wakuorg/nwaku:v0.22.0-rc.0}
```

**Review by SionoiS — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (gabrielmer) [reply docker-compose.yml:34]: It works for me, I think that `docker.io` gets automatically resolved, and the `:-` makes `wakuorg/nwaku:v0.22.0-rc.0` the default value of `NWAKU_IMAGE` but if another value is set as env variable then the env variable has precedence.

Anyway, don't think it matters much but lmk what you think

Comment (SionoiS) [reply docker-compose.yml:34]: leave it doesn't matter I guess.

**Pull request merged**